pkg/apis/autoscaling/v1alpha1/pa_lifecycle_test.go

```diff
@@ -469,9 +470,12 @@ func TestMarkResourceNotOwned(t *testing.T) {
 }
 
 func TestMarkResourceFailedCreation(t *testing.T) {
-	pa := pa(map[string]string{})
-	pa.Status.MarkResourceFailedCreation("doesn't", "matter")
-	active := pa.Status.GetCondition("Active")
+	pa := &PodAutoscalerStatus{}
+	pa.MarkResourceFailedCreation("doesn't", "matter")
+	d := (*duckv1beta1.Status)(pa)
```

Comment:
I had to do this too.  Given the explosion this is having, perhaps a method is called for (it can be private).

Reply:
I am not against doing this, but I'd like to do this on its own cadence.